tests/NLog.UnitTests/Targets/FileTargetTests.cs

```diff
@@ -1562,7 +1562,7 @@ public void FileTarget_LogAndArchiveFilesWithSameName_ShouldArchive()
         }
 
         [Fact]
-        public void Single_Archive_File_Rolls_Correctly()
+        public void SingleArchiveFileRollsCorrectly()
```

Comment:
don't mind the snake cases in the test names. If there are in the test cases, it's fine IMO